Gemfile

```diff
@@ -35,7 +36,9 @@ group :development, :test do
   gem "dotenv-rails"
   gem "faker"
   gem "mdl"
+  gem "rubocop-factory_bot", "2.25.1"
```

Comment:
You can remove this line after you update the rest of the gems to make it cleaner

```suggestion
```